Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.py

```diff
@@ -538,16 +543,39 @@ def fetch_incidents(client, max_results, last_run, first_fetch, filters):
     return next_run, incidents
 
 
+def params_to_filter(severity, resolution_status):
+    filters: Dict[str, Any] = {}
+    if len(severity) == 1:
+        filters['severity'] = {'eq': SEVERITY_OPTIONS[severity[0]]}
+    else:
+        severities = []
+        for severity_option in severity:
+            severities.append(SEVERITY_OPTIONS[severity_option])
+        filters['severity'] = {'eq': severities}
+    if len(resolution_status) == 1:
+        filters['resolutionStatus'] = {'eq': RESOLUTION_STATUS_OPTIONS[resolution_status[0]]}
+    else:
+        severities = []
```

Comment:
```suggestion
        resolution_statuses = []
```

The same change in this else block